### PR TITLE
Feat : Support for Duration in SnackbarConfig

### PR DESCRIPTION
**Pull request by vishhmakasana**
Duration field added in snackbar_config.dart so that we will not have to specify Duration on each show snackbar call.

### CODE REVIEW

File: packages/stacked_services/lib/src/snackbar/snackbar_service.dart [hunk 1]
```diff
@@ -160,6 +160,9 @@ class SnackbarService {
               textAlign: snackbarConfig.titleTextAlign,
             )
           : snackbarConfig.titleText ?? null,
+      /// Please check if its intentionally or some typo mistake : it should be
+      // messageText: snackbarConfig.messageColor != null || message.isNotEmpty
```
Comment (Eimen2018) [inline packages/stacked_services/lib/src/snackbar/snackbar_service.dart]: Yes i think it should be snackbarConfig.messageColor

**Review by Eimen2018 — CHANGES_REQUESTED** [review]
LGTM, please update the comment and add package maintenance (increase version and update change log)

**Review by FilledStacks — APPROVED** [review]
Thanks for the PR and Thanks @Eimen2018 for the review.

We have an automatic deploy and update of changelog and version now 😄

**Review by FilledStacks — CHANGES_REQUESTED** [review]
Only request is to fix merge issues and then we can move forward

**Pull request closed without merging**